docs/source/admin/traffic_vault.rst

```diff
@@ -29,6 +29,28 @@ Configuring Traffic Vault
 =========================
 The following steps were taken to configure Riak in our environments.
 
+
+Self Signed Certificate configuration
+-------------------------------------
+Below shows how to create self signed certs for Trafficvault.
+
+Note: Self Signed Certificates are not recommended for production use. Intended for dev or learning purposes only. Modify subject as necessary.   
+
+	``cd ~``
+	``mkdir certs``
+	``cd certs``
+	``openssl genrsa -out ca-bundle.key 2048``
+	``openssl req -new -key ca-bundle.key -out ca-bundle.csr -subj "/C=US/ST=CO/L=DEN/O=somecompany/OU=CDN/CN=somecompany.net/emailAddress=someuser@somecompany.net"``
+	``openssl x509 -req -days 365 -in ca-bundle.csr -signkey ca-bundle.key -out ca-bundle.crt``
+	``openssl genrsa -out server.key 2048``
+	``openssl req -new -key server.key -out server.csr -subj "/C=US/ST=CO/L=DEN/O=somecompany/OU=CDN/CN=somecompany.net/emailAddress=someuser@somecompany.net"``
+	``openssl x509 -req -days 365 -in server.csr -CA ca-bundle.crt -CAkey ca-bundle.key -CAcreateserial -out server.crt``
+	``mkdir /etc/riak/certs``
+	``mv -f server.crt /etc/riak/certs/.``
+	``mv -f server.key /etc/riak/certs/.``
+	``mv -f ca-bundle.crt /etc/pki/tls/certs/.``
+
```

Comment:
Instead of a bunch of separately styled lines, can you use the `code` directive? The syntax is:
```rst
.. code:: <syntax>
    :caption: <caption>

    Content. Note the blank line between the caption and the content, as well as the space between the
    syntax and the 'code' directive - both are syntactically important. 
    These content blocks (and the capiton) should be indented (TABS!!!), but the '..' part of the 'code' directive should
    not have any whitespace between it and the beginning of the line.
    There must be a blank line between the end of the content and the start of anything else

This text won't be part of the code block
```
This will make the code block directly linkable and reference-able, and will add syntax highlighting. Yours is a shell script, so you'd probably want to highlight it as 'bash' or 'shell', and it'd look something like this:
```rst
.. code-block:: shell
    :caption: Self-Signed Certificate Configuration

    cd ~
    mkdir certs
    etc.
```
You may at that point decide this doesn't need a subsection heading that just copies the caption, but that's really up to you.

Reply:
Certainly! I can do that.. 

Would this be more of what we're looking for? 

Self Signed Certificate configuration
-------------------------------------

note:: Self Signed Certificates are not recommended for production use. Intended for dev or learning purposes only. Modify subject as necessary.

`Self Signed Certificate configuration
-------------------------------------

note:: Self Signed Certificates are not recommended for production use. Intended for dev or learning purposes only. Modify subject as necessary.

code-block:: shell
    :caption: Self-Signed Certificate Configuration

        cd ~
        mkdir certs
        cd certs
        openssl genrsa -out ca-bundle.key 2048
        openssl req -new -key ca-bundle.key -out ca-bundle.csr -subj "/C=US/ST=CO/L=DEN/O=somecompany/OU=CDN/CN=somecompany.net/emailAddress=someuser@somecompany.net"
        openssl x509 -req -days 365 -in ca-bundle.csr -signkey ca-bundle.key -out ca-bundle.crt
        openssl genrsa -out server.key 2048
        openssl req -new -key server.key -out server.csr -subj "/C=US/ST=CO/L=DEN/O=somecompany/OU=CDN/CN=somecompany.net/emailAddress=someuser@somecompany.net"
        openssl x509 -req -days 365 -in server.csr -CA ca-bundle.crt -CAkey ca-bundle.key -CAcreateserial -out server.crt
        mkdir /etc/riak/certs
        mv -f server.crt /etc/riak/certs/.
        mv -f server.key /etc/riak/certs/.
        mv -f ca-bundle.crt /etc/pki/tls/certs/.
`

Reply:
I believe so. It's a little hard to tell, because GitHub is trying to format it as markdown, but it looks roughly right.

Reply:
changing that in github.. one moment.

Reply:
Okay, that looks good.